spec/date_time/offsets_spec.rb

```diff
@@ -266,6 +267,16 @@
           DateTime.new(2015,1,31))
       end
     end
+
+    context "#on_offset?" do
+      it "returns false if date is on the offset" do
```

Comment:
The date you've specified is not on the offset so should this test not be `"returns false if date is not on the offset"`?